### PR TITLE
Add flink-0.10.1 system

### PR DESCRIPTION
**Pull request by carabolic**
The flink build with support for Hadoop 2.7 and Scala 2.10 is chosen.

### CODE REVIEW

File: peel-extensions/src/main/resources/reference.flink-0.10.1.conf [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# include common flink configuration
+include "reference.flink.conf"
+
+system {
+    flink {
+        path {
+            archive.url = "http://archive.apache.org/dist/flink/flink-0.10.1/flink-0.10.1-bin-hadoop27-scala_2.10.tgz"
```
Comment (aalexandrov) [inline peel-extensions/src/main/resources/reference.flink-0.10.1.conf]: I can see that there is a version for hadoop2:

http://archive.apache.org/dist/flink/flink-0.10.1/flink-0.10.1-bin-hadoop2-scala_2.10.tgz

why don't we link this one?

Comment (carabolic) [inline peel-extensions/src/main/resources/reference.flink-0.10.1.conf]: @aalexandrov based on the documentation from the [Flink download page](http://flink.apache.org/downloads.html) this seems to be for Hadoop 2.3.0. I can change it though.

Comment (aalexandrov) [inline peel-extensions/src/main/resources/reference.flink-0.10.1.conf]: @carabolic I suggest to go for that, this should give us the greatest compatibility across all Hadoop 2.x versions.

**Comment by carabolic** [comment]
I've updated the PR to use `flink-*-bin-hadoop2-*.tgz` instead of `flink-*-bin-hadoop27-*.tgz` now.

**Comment by carabolic** [comment]
Fixed with https://github.com/stratosphere/peel/commit/3124ab52966722dac8c055accd062af959026351

**Pull request closed without merging**